fluent-bit/fluent-bit-go-loki/Makefile

```diff
@@ -0,0 +1,14 @@
+all: test
```

Comment:
Any reason this is a separate file? Could it be integrated into the main Makefile?

Reply:
Oh. Your comments is reasonable.
Hm, how about remove this file in the first step?

I'm struggling to understand Loki Makefile structure. 😅

Reply:
Okay, no problem:

`all`: remove
`fast`: add to current makefile, rename something like `fluent-bit`
`test`: remove, already covered by root `make test`
`dep`: remove
`clean`: add the command to the current `make clean`, note to prefix the wildcards with the correct directory

Reply:
Wow. It's super fast advise!

I've modified the current makefile as follows:

`all`: removed
`fast`: renamed to `fluent-bit` and use `go build -buildmode=c-shared -o out_loki.so .` command
`test`: removed
`dep`: removed
`clean`: Use concrete file name. Without wildcards.

Reply:
Nice! Could you integrate those targets into the `/Makefile`?

Reply:
Sorry, I'm still not to be able to integrate them into the `/Makefile`...

Reply:
Okay, no problem. May I help out?

Reply:
Okay, below is a patch that includes the changes.

Save to a file, apply using `git am < filename`.

```patch
From dfc82f917839704a367b7b98c887fd3295ec57aa Mon Sep 17 00:00:00 2001
From: sh0rez <me@shorez.de>
Date: Mon, 5 Aug 2019 12:12:46 +0200
Subject: [PATCH] refactor(fluent-bit): move make targets into root Makefile

---
 Makefile                               | 7 +++++++
 fluent-bit/fluent-bit-go-loki/Makefile | 5 -----
 2 files changed, 7 insertions(+), 5 deletions(-)
 delete mode 100644 fluent-bit/fluent-bit-go-loki/Makefile

diff --git a/Makefile b/Makefile
index c05b5e4b..44074406 100644
--- a/Makefile
+++ b/Makefile
@@ -210,6 +210,7 @@ clean:
 	rm -rf .cache
 	rm -rf cmd/docker-driver/rootfs
 	rm -rf dist/
+	rm -rf fluent-bit/fluent-bit-go-loki/out_loki.*
 	go clean ./...
 
 #########
@@ -333,6 +334,12 @@ docker-driver-clean:
 	-docker plugin rm grafana/loki-docker-driver:$(IMAGE_TAG)
 	rm -rf cmd/docker-driver/rootfs
 
+#####################
+# fluent-bit plugin #
+#####################
+fluent-bit-plugin:
+	go build -buildmode=c-shared -o fluent-bit/fluent-bit-go-loki/out_loki.so ./fluent-bit/fluent-bit-go-loki
+
 ########################
 # Bigtable Backup Tool #
 ########################
diff --git a/fluent-bit/fluent-bit-go-loki/Makefile b/fluent-bit/fluent-bit-go-loki/Makefile
deleted file mode 100644
index b3a25f36..00000000
--- a/fluent-bit/fluent-bit-go-loki/Makefile
+++ /dev/null
@@ -1,5 +0,0 @@
-fluent-bit:
-	go build -buildmode=c-shared -o out_loki.so .
-
-clean:
-	rm -rf out_loki.so out_loki.h
-- 
2.17.1

```

Reply:
Thanks! I've confirmed your patch is working! 😃